### ISSUE TITLE
Add ca cert location and password-flags when creating a new connection. 

### ISSUE THREAD

**Issue by ryanvade** [seveas/python-networkmanager#21]
How can I set the location of a ca certification and pasword-flags when creating a new wpa-enterprise connection? 
Based on your example:

```
WPA_connection = {
     '802-11-wireless': {'mode': 'infrastructure',
                         'security': '802-11-wireless-security',
                         'ssid': 'n-m-example-connection'},
     '802-11-wireless-security': {'auth-alg': 'open', 'key-mgmt': 'wpa-eap'},
     '802-1x': {'eap': ['peap'],
                'ca-cert': cert_location,
                'identity': eid,
                'pasword-flags': '1',
                'phase2-auth': 'mschapv2'},
     'connection': {'id': 'nm-example-connection',
                    'type': '802-11-wireless',
                    'uuid': str(uuid.uuid4())},
     'ipv4': {'method': 'auto'},
     'ipv6': {'method': 'auto'}
}
NetworkManager.Settings.AddConnection(WPA_connection)
```

but that gives me a dbus exception:

```
dbus.exceptions.DBusException: org.freedesktop.NetworkManager.Settings.Connection.InvalidProperty: 802-1x.ca-cert: can't set property of type 'GBytes' from value of type 's'
```

*EDIT: Also, how do you set the mac-address?

**Comment by seveas** [comment]
From the error I'm guessing you are using python 3 and should be converting the string to a bytes object.

**Comment by ryanvade** [comment]
I will try that.  But am I doint the rest correctly?

**Comment by ryanvade** [comment]
Encoding is not working either:
```
'ca-cert': cert_location.encode(encoding='utf-8'),
```
gives the same error.

**Comment by ryanvade** [comment]
Okay, according to this page:
https://developer.gnome.org/NetworkManager/0.9/ref-settings.html
ca-cert requires a bytearray with a prefix of file:// and ending with a NULL character. So I created the string with the requirements, cert_location, then I did this:
```
cert_btArray = bytearray(source=cert_location, encoding='utf-8')
```
But I am still getting this error:
dbus.exceptions.DBusException: org.freedesktop.NetworkManager.Settings.Connection.InvalidProperty: 802-1x.ca-cert: can't set property of type 'GBytes' from value of type 'ai'

**Comment by yon2004** [comment]
I am also having this same issue.

**Comment by seveas** [comment]
Finally fixed in 91ab5368
